dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -1010,16 +1012,16 @@ public void updateMenuLinks(Host workinghost,Host updatedhost) throws DotDataExc
     private PaginatedArrayList<Host> search(String filter, String condition, boolean showSystemHost, int limit, int offset, User user, boolean respectFrontendRoles) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "condition". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)